addons/mail/static/tests/message/message.test.js

```diff
@@ -202,7 +202,8 @@ test("Can edit message comment in chatter", async () => {
     await contains(".o-mail-Message-content", { text: "edited again (edited)" });
 });
 
-test("Can edit message comment in chatter (mobile)", async () => {
+test.tags("mobile");
+test.debug("Can edit message comment in chatter (mobile)", async () => {
```

Comment:
i dont see it, i dont see it, i dont see it

Reply:
no, i meant, the .debug, haha

Reply:
Oh... I'm so blind, did you see something? 🐵